tests/Main.test.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
+import 'react-native-gesture-handler/jestSetup'
```

Comment:
shall we move this to a global test setup file, similarly to what's described [here](https://reactnavigation.org/docs/testing/) (first google result but looks like a good approach)?

Reply:
Yeah, I was planning on creating a `jest.setup.js`. I hadn't done it until this point because I was working within the one test file. But I'll add it now.